### PR TITLE
fix wrong url in zh docs components.md

### PR DESCRIPTION
**Pull request by B1F030**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
fix wrong url in zh docs

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

### CODE REVIEW

File: i18n/zh/docusaurus-plugin-content-docs/current/core-concepts/components.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Karmada API 服务器是直接使用 Kubernetes 的 kube-apiserver 实现的，
 kube-controller-manager 由一组控制器组成，Karmada 只是从 Kubernetes 的官方版本中挑选了一些控制器，以保持与原生控制器一致的用户体验和行为。
 
 值得注意的是，并非所有的原生控制器都是 Karmada 所需要的，
-推荐的控制器请参阅 [Recommended Controllers](../administrator/configuration/configure-controllers#required-controllers)。
+推荐的控制器请参阅 [Recommended Controllers](../administrator/configuration/configure-controllers#必需的控制器)。
```
Comment (windsonsea) [inline i18n/zh/docusaurus-plugin-content-docs/current/core-concepts/components.md:32]:
```suggestion
推荐的控制器请参阅[推荐的控制器](../administrator/configuration/configure-controllers/#必需的控制器)。
```

**Review by windsonsea — COMMENTED** [review]
The [preview](https://deploy-preview-645--karmada.netlify.app/zh/docs/core-concepts/components) still doesn't work for these links.

Please add your signature by running:

```
git commit -s --amend
git push -f
```

File: i18n/zh/docusaurus-plugin-content-docs/current/core-concepts/components.md (hunk 1 above)
Comment (B1F030) [reply i18n/zh/docusaurus-plugin-content-docs/current/core-concepts/components.md]: Thanks. Done.

**Comment by windsonsea** [comment]
奇怪，预览中这些 link 都不对：
https://deploy-preview-645--karmada.netlify.app/zh/docs/core-concepts/components/

**Comment by B1F030** [comment]
> 奇怪，预览中这些 link 都不对： https://deploy-preview-645--karmada.netlify.app/zh/docs/core-concepts/components/

我这边测试好像正常呀，next版本中，都能正常链接到中文文档的对应章节

**Comment by windsonsea** [comment]
Thanks.
/lgtm

**Review by RainbowMango — APPROVED** [review]
/approve

**Pull request merged**